content/docs/command-reference/cache/dir.md

```diff
@@ -61,6 +61,9 @@ $ cat .dvc/config
 `../dir` has been resolved relative to the `.dvc/` dir, resulting in
 `../../dir`.
 
+The command `dvc cache dir` resolves your relative paths so that they work with
+the location of the config file in your project.
+
```

Comment:
No need for this, it's been explained in the description as mentioned in https://github.com/iterative/dvc.org/pull/1316#pullrequestreview-416473218. Please revert